logic_tree_tools.py

```diff
@@ -592,10 +605,12 @@ def get_logic_tree_pairs_for_individual_ground_motion_models(
         If more than one tectonic_region_type is included in any tectonic_region_type_set.
     """
 
-    if len(tectonic_region_type_sets[0]) == 1:
-        raise ValueError(
-            "Only one tectonic_region_type can be included in each tectonic_region_type_set passed to this function."
-        )
+    print()
+
+    # if len(tectonic_region_type_sets[0]) == 1:
+    #     raise ValueError(
+    #         "Only one tectonic_region_type can be included in each tectonic_region_type_set passed to this function."
+    #     )
```

Comment:
Can probably tidy this up

Reply:
sorry, this was all commented out for debugging. Uncommented now.